### PR TITLE
Tagline branch pushed

### PR DESCRIPTION
**Pull request by Daniel-Dudek**

### CODE REVIEW

**Review by AdrianNQ99 — DISMISSED** [review]
muy bien!!!

File: website/templates/tagline.html [hunk 1]
```diff
@@ -1,3 +1,8 @@
 <!-- your html code here -->
 <!-- Also called a hero image or a jumbotron before bootstrap 5. This might be helpful: https://getbootstrap.com/docs/5.1/examples/jumbotron/ -->
```
Comment (ScarzoJ) [inline website/templates/tagline.html]: Remove boilerplate example

**Review by ScarzoJ — CHANGES_REQUESTED** [review]

File: website/templates/tagline.html (hunk 1 above)
Comment (Daniel-Dudek) [reply website/templates/tagline.html]: Boilerplate example has been removed and pushed

**Review by ScarzoJ — APPROVED** [review]

**Review by PilarGallegoS — APPROVED** [review]
Muy bien!

**Pull request merged**